### DISCUSSION TITLE
Help understanding PHOTOPRISM_AUTO_IMPORT

### DISCUSSION THREAD

**Discussion by andrewlow** [photoprism/photoprism#879] [Q&A] [answered]
I'm running using the Pure Docker approach - I'm pretty comfortable with docker as a whole and recognize this is a more advanced way to do things, but I don't believe that is related to what's happening.

I can see (and have confirmed) that the environment variable is set in the container

PHOTOPRISM_AUTO_IMPORT=300

This is the default.

When I use a Finder window on OSX connected to the WebDAV 'import' folder -- this allows me to easily copy files from my OSX machine to the import directory.

On my local machine hosting photoprism - if I visit the URL `http://localhost:2342/library/import` - I've also checked off the box 'Move Files'

![image](https://user-images.githubusercontent.com/2952475/104199996-20a9ef00-53f6-11eb-8c1e-42420131f8be.png)

It does appear that if I copy files to the import folder (via WebDAV) -- if I wait ~5min, then the pictures are automatically imported into my library.

However - the files are not 'Move'd (ie: deleted from the import directory) unless I click on the `Import` button in the screen capture above.

This is confusing. Is there another configuration option I need to setup? I'd really like auto-import to work just like clicking on the Import button.

**Comment by lastzero** [comment, marked as answer]
You're right - we've decided to launch this new feature in a conservative way. Next build / release will use the setting from the Web UI.

**Reply by andrewlow** [reply]
Thanks. I'm looking forward to the next release.

**Reply by andrewlow** [reply]
Oh - and exposing the 'Move' as a env var flag might be a good idea. It may also be a good idea for consistency to allow all configurations managed in the same way - (ie: config file / env var / web UI can all manage the values)

**Reply by lastzero** [reply]
Your feature request was implemented yesterday, so you don't need to wait. Also you can change all values in options.yml, settings.yml or as command line parameter if you prefer that.

We may also implement everything in the UI although some settings like the base folder should not be changed while the system is running. Also note that most of the work is translating the UI to all the languages and writing proper docs for them. Otherwise inexperienced users will make mistakes and we are getting blamed on top of providing this for free.
